web/src/components/core/NotificationMark.jsx

```diff
@@ -38,5 +40,23 @@ import React from "react";
  * @param {string} props.label - the label to be announced by screen readers
  */
 export default function NotificationMark ({ label }) {
-  return <span className="notification-mark" role="status" aria-label={label} />;
+  const [hasIssues, setHasIssues] = useState(false);
+  const { issues: client } = useInstallerClient();
+  const { cancellablePromise } = useCancellablePromise();
+
+  const checkIssues = useCallback(async () => {
+    setHasIssues(await cancellablePromise(client.any()));
+  }, [client, cancellablePromise, setHasIssues]);
+
+  useEffect(() => {
+    checkIssues();
+    return client.onIssuesChange(checkIssues);
+  }, [client, checkIssues]);
+
+  return (
+    <If
+      condition={hasIssues}
```

Comment:
Thinking aloud:

Maybe in the future we will display the mark because whatever other reason, not only for issues. If so, a way to know the reasons why there are notifications will be needed too.

But for me this is a very good start because it's the only use case we have by now. We can improve it as soon as we have another reason for displaying such a UI element.

Thus, I guess we can use the component in the "Show Issues" link for displaying the mark close to it too. CSS will need adjustments, though.